### PR TITLE
Test for efficient implementation

### PR DESCRIPTION
**Pull request by fulldecent**

### CODE REVIEW

File: test_data/encoding.csv [hunk 1]
```diff
@@ -127,3 +127,16 @@
 68.3500147997595,113.625636875353,15,9PWM9J2G+272FWJV
 38.1176000887231,165.441989844555,15,8VC74C9R+2QX445C
 -28.1217794010122,-154.066811473758,15,5337VWHM+77PR2GR
+#
+# Test for poor performance, which may be a security issue (denial of service)
+# see https://github.com/google/open-location-code/issues/373
+# and https://github.com/google/open-location-code/pull/370
+-80.000000,-170.000000,2,22000000+
+-80.000000,-36170.000000,2,22000000+
```
Comment (bilst) [inline test_data/encoding.csv:135]: NIT: Would the first and last case suffice?

**Review by bilst — COMMENTED** [review]
Nice test--thanks! Unfortunately though the java and python implementations still have the performance issue so would need fixing before adding it.

File: test_data/encoding.csv (hunk 1 above)
Comment (fulldecent) [reply test_data/encoding.csv]: Yes. Updated at a89ce4cbf729348c06842bf81bb3eb9c80fa1df3

**Comment by fulldecent** [comment]
This PR is included inside (actually, obviated by) https://github.com/google/open-location-code/pull/463

And additionally, specification is added:

> Longitude inputs MUST NOT cause runtime performance linearly dependent on the longitude. E.g. use `LONGITUDE % 360`, not `WHILE (LONGITUDE > 180) LONGITUDE -= 360`.

**Comment by drinckes** [comment]
I'm going to close without merging since it sounds like this is already addressed?

**Comment by fulldecent** [comment]
I see that the implementations are now compliant with this test. 

But please review and think about merging because this is a valid test case that we should be testing for across all implementations now and in the future. 

@drinckes

**Pull request closed without merging**